network/docker-compose-solo.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
+
+services:
+  thor-solo:
+    container_name: thor-solo
+    image: ${THOR_IMAGE:-vechain/thor:latest}
```

Comment:
is there any way to build latest thor image locally ? as this one is taking the old image

if not maybe we can update readme how to do it locally